traveler-swift-core/TravelerKit/Models/ProductItemCategory.swift

```diff
@@ -17,4 +17,6 @@ public enum ProductItemCategory: String, Decodable {
     case show = "Show"
     case event = "Event"
     case nightlife = "Nightlife"
+    case parking = "Parking"
```

Comment:
It seems only booking products have `categories` property while parking products don't. Why is there a `ProductItemCategory` case `parking`?

Reply:
That category is there because the backend has not finished the refactoring regarding product categories. Parking is being surfaced as a category at the moment and if its not there in that enum the parsing will break.